returnn/datasets/meta.py

```diff
@@ -1406,13 +1407,19 @@ def __init__(
           Now it could happen that ceildiv(data_len1 + data_len2, 6) < align_len1 + align_len2.
           This option would repeat intermediate ending frames such that data_len1 % 6 == 0,
           by setting it to {"data": 6}.
+        :param dict[str,(str,int)]|None pad_truncate_data_to_multiple_of: data_key -> (target_key, multiple).
+            Force data_len == target_len * multiple for specified data and target.
+            Similar to repeat_in_between_last_frame_up_to_multiple_of but also truncates the data.
```

Comment:
> Similar to repeat_in_between_last_frame_up_to_multiple_of but also truncates the data.

This does not really explain it. In what case would it be relevant to truncate the data?

Reply:
I know, this is not the final version as mentioned in the commit comment. I want to first consider your long comment above that I haven't read through, maybe there is indeed situations(especially with padding) that is not covered by my code.